e2e/support/helpers/e2e-ui-elements-helpers.js

```diff
@@ -1,6 +1,6 @@
 // various Metabase-specific "scoping" functions like inside popover/modal/navbar/main/sidebar content area
 export const POPOVER_ELEMENT =
-  ".popover[data-state~='visible'],[data-position]:not(.emotion-HoverCard-dropdown)";
+  ".popover[data-state~='visible'],[data-element-id=mantine-popover]";
```

Comment:
`[data-element-id=mantine-popover]` is basically the same as `[data-position]`, but the reason we can remove `:not(.emotion-HoverCard-dropdown)` now is because we don't display hover card before we hover over them now.

If you read the issue description this was the cause of the flake, because the fake columns somehow have a different class name `.mantine-HoverCard-dropdown` instead of `.emotion-HoverCard-dropdown`. And I don't want to keep adding more class since it looks more like a hack.